src/App.js

```diff
@@ -1,23 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Firebase from './pages/firebase';
```

Comment:
This needs to be upper-case `/Pages/` or the app doesn't start for me and in the CI pipeline. Fixing this should fix the issue found [here](https://github.com/the-collab-lab/tcl-41-smart-shopping-list/runs/5853803455?check_suite_focus=true)